systemtests/tests/py3plug-fd-mariabackup/testrunner

```diff
@@ -17,6 +17,18 @@ export TestName
 
 skip_if_root
 
+# skip_if_suse()
+# on SUSE before bareos 24, mariabackup plugin will use the
+# python mysqlclient module and will try by default to connect the
+# global instance.
+# As we don't want to replace any ~/.my.cnf we simply skip the test.
+if source /etc/os-release; then
+  if echo "${PRETTY_NAME}" | grep -q "SUSE"; then
+    echo "${TestName} test skipped: SUSE platform detected."
+    exit 77;
+  fi
+fi
+
```

Comment:
I would propose that we do this in cmake instead.

Here is a way with which you can find out if you are on suse or not:
```
find_program(LSB_RELEASE_EXEC lsb_release)

if (${LSB_RELEASE_EXEC} STREQUAL "LSB_RELEASE_EXEC-NOTFOUND")
  message("not found, so not suse")
else()
  execute_process(COMMAND ${LSB_RELEASE_EXEC} -is
    OUTPUT_VARIABLE LSB_RELEASE_ID_SHORT
    OUTPUT_STRIP_TRAILING_WHITESPACE
  )

  if (${LSB_RELEASE_ID_SHORT} STREQUAL "SUSE")
    message("we are on suse!")
  endif()
endif()
```

What do you think ?

Reply:
This can be added to `py3plug-fd-mariabackup/CMakeLists.txt`

Reply:
I will check if all of our builder have lsb_release installed. os-release is always present with systemd. 
I don't know if switching 4 lines of bash to 13 lines of cmake is an improvment (I found the solutiion more elegant, but as one disadvantage: the test will not be setup for any of suse, and as such nobody won't be able to quickly try fix it.

Reply:
Not sure, if this is still used, but cmake already contains something similar:

- `core/cmake/BareosGetDistInfo.cmake`
- `core/cmake/distname.sh`